metric/system/diskio/diskstat.go

```diff
@@ -48,6 +48,7 @@ type IOMetric struct {
 }
 
 // IOStat carries disk statistics for all devices
+// nolint: nolintlint // platform diffs
```

Comment:
`nolint: nolintlint` this doesn't seem like a typo?

Reply:
No, it's not a typo. I have to turn off warnings for `nolint` for the next line because on Darwin the `structcheck` linting does not apply.